### DISCUSSION TITLE
The game crashed whilst initializing game Error: java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'ad astra'!

### DISCUSSION THREAD

**Discussion by yvngkza** [FabricMC/fabric-api#3081] [Player support]
idk what is the problem here :(

---- Minecraft Crash Report ----
// This doesn't make any sense!

Time: 26/5/23 15:55
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'ad_astra'!
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:53)
	at net.minecraft.class_310.<init>(class_310.java:452)
	at net.minecraft.client.main.Main.main(Main.java:197)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.BootstrapMethodError: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_970 failed
	at com.github.alexnijjar.ad_astra.client.registry.ClientModEntities.register(ClientModEntities.java:38)
	at com.github.alexnijjar.ad_astra.client.AdAstraClient.onInitializeClient(AdAstraClient.java:61)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
	... 7 more
Caused by: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_970 failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:427)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:112)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 10 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422)
	... 14 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method polymer_changeArmorTexture(Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;Lnet/minecraft/class_1309;Lnet/minecraft/class_1304;ILnet/minecraft/class_572;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;Lnet/minecraft/class_1799;)V in polymer.mixins.json:client.compat.armor_ArmorFeatureRendererMixin from mod polymer failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap polymer-refmap.json
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:468)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1385)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1062)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:402)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:327)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	... 17 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:53)
	at net.minecraft.class_310.<init>(class_310.java:452)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:API base de Windows 32 avanzado:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:Biblioteca de controles de la experiencia del usuario:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Crypto API32:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.19041.867 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DLL de API de cliente DNS:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.19041.2130 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.19041.2673 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:API auxiliar para IP:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Archivo DLL de cliente API BASE de Windows NT:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Archivo DLL de cliente API BASE de Windows NT:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		MpOav.dll:IOfficeAntiVirus Module:4.18.2304.8 (WinBuild.160101.0800):Microsoft Corporation
		NLAapi.dll:Network Location Awareness 2:10.0.19041.2193 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.19041.610 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.19041.320 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE para Windows:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		OleAut32.dll:OLEAUT32.DLL:10.0.19041.985 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Ayudante de los datos de rendimiento DLL de Windows:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Tiempo de ejecución de llamada a procedimiento remoto:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Archivo DLL común del shell de Windows:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:Archivo DLL de cliente API USER de Windows multiusuario:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.29.30133.0 built by: vcwrkspc:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Servicios HTTP de Windows:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:DLL de 32 bits de Windows Socket 2.0:10.0.19041.1081 (WinBuild.160101.0800):Microsoft Corporation
		WSOCK32.dll:Windows Socket 32-Bit DLL:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		Wldp.dll:Directiva de bloqueo de Windows:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.19041.2075 (WinBuild.160101.0800):Microsoft Corporation
		apphelp.dll:Biblioteca de compatibilidad de aplicaciones cliente:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		awt.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		bcrypt.dll:Biblioteca de primitivas criptográficas de Windows:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.19041.2486 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM para Windows:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.19041.2788 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc.DLL:Servicio cliente DHCP:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc6.DLL:Cliente DHCPv6:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		fwpuclnt.dll:API de modo usuario de FWP/IPsec:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.19041.2788 (WinBuild.160101.0800):Microsoft Corporation
		glfw.dll
		java.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		javaw.exe:OpenJDK Platform binary:17.0.1.0:Microsoft
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		jli.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		jna11465918611004581251.dll:JNA native library:6.1.2:Java(TM) Native Access (JNA)
		jvm.dll:OpenJDK 64-Bit server VM:17.0.1.0:Microsoft
		kernel.appcore.dll:AppModel API Host:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		management.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		management_ext.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		mdnsNSP.dll:Bonjour Namespace Provider:3,1,0,1:Apple Inc.
		msvcp140.dll:Microsoft® C Runtime Library:14.29.30133.0 built by: vcwrkspc:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.19041.789 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Proveedor de servicios de Microsoft Windows Sockets 2.0:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:Proveedor de correcciones de compatibilidad (shim) de nomenclaturas de correo electrónico:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Enrutador de Windows NCrypt:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		nio.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		ntdll.dll:DLL del nivel de Windows NT:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:DLL de objetos de rendimiento del sistema Windows:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:Proveedor de espacio de nombres PNRP:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.19041.844 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.19041.320 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.19041.320 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Biblioteca de utilidades de Shell:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		svml.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.19041.789 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.29.30133.0 built by: vcwrkspc:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		win32u.dll:Win32u:10.0.19041.2846 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:API de almacenamiento de Microsoft WinRT:10.0.19041.2965 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
Stacktrace:
	at net.minecraft.client.main.Main.main(Main.java:197)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.18.2
	Minecraft Version ID: 1.18.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.1, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 4298688400 bytes (4099 MiB) / 6174015488 bytes (5888 MiB) up to 6442450944 bytes (6144 MiB)
	CPUs: 4
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 3 2200G with Radeon Vega Graphics    
	Identifier: AuthenticAMD Family 23 Model 17 Stepping 0
	Microarchitecture: Zen
	Frequency (GHz): 3,49
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 4
	Graphics card #0 name: AMD Radeon(TM) Vega 8 Graphics
	Graphics card #0 vendor: Advanced Micro Devices, Inc. (0x1002)
	Graphics card #0 VRAM (MB): 2048,00
	Graphics card #0 deviceId: 0x15dd
	Graphics card #0 versionInfo: DriverVersion=31.0.14043.1000
	Memory slot #0 capacity (MB): 4096,00
	Memory slot #0 clockSpeed (GHz): 2,40
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 4096,00
	Memory slot #1 clockSpeed (GHz): 2,40
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 14593,69
	Virtual memory used (MB): 12465,73
	Swap memory total (MB): 8501,18
	Swap memory used (MB): 0,00
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx6G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		ad_astra: Ad Astra 1.0.9
			team_reborn_energy: Energy 2.2.0
		additionalbanners: AdditionalBanners 7.0.8
		adorn: Adorn 3.5.0+1.18.2
		ambientsounds: Ambiend Sounds (Fabric) 5.0.16
		appleskin: AppleSkin 2.4.1+mc1.18.2
		architectury: Architectury 4.10.88
		artifacts: Artifacts 7.0.3+fabric
			com_github_llamalad7_mixinextras: MixinExtras 0.0.7
			expandability: ExpandAbility 6.0.0
			step-height-entity-attribute: Step Height Entity Attribute 1.0.0
		bclib: BCLib 1.4.7
		betterend: Better End 1.1.1
		betterfortresses: YUNG's Better Nether Fortresses 1.18.2-Fabric-1.0.0
			org_reflections_reflections: reflections 0.10.2
		bettermineshafts: YUNG's Better Mineshafts 1.18.2-Fabric-2.2
		betternether: Better Nether 6.1.2
		betterstrongholds: YUNG's Better Strongholds 1.18.2-Fabric-2.1.1
		betterthirdperson: Better Third Person 1.8.1
		bettervillage: Better village 3.0.0
		betterwitchhuts: YUNG's Better Witch Huts 1.18.2-Fabric-1.0.1
		bookshelf: Bookshelf 13.2.53
		bosses_of_mass_destruction: Bosses of Mass Destruction (Beta) 1.3.9-1.18.2
			maelstrom_library: Maelstrom Library 1.2.4-1.18.2
			multipart_entities: MultipartEntities 1.1.5-1.18.2
		cardinal-components: Cardinal Components API 4.2.0
			cardinal-components-base: Cardinal Components API (base) 4.2.0
			cardinal-components-block: Cardinal Components API (blocks) 4.2.0
			cardinal-components-chunk: Cardinal Components API (chunks) 4.2.0
			cardinal-components-entity: Cardinal Components API (entities) 4.2.0
			cardinal-components-item: Cardinal Components API (items) 4.2.0
			cardinal-components-level: Cardinal Components API (world saves) 4.2.0
			cardinal-components-scoreboard: Cardinal Components API (scoreboard) 4.2.0
			cardinal-components-world: Cardinal Components API (worlds) 4.2.0
		cloth-config: Cloth Config v6 6.2.62
			cloth-basic-math: cloth-basic-math 0.6.1
		collective: Collective 6.53
		craftabletotemofundying: Craftable Totem Of Undying 1.18.2-1.5.2-[FABRIC]
		creativecore: CreativeCore (Fabric) 2.6.12
		croptopia: Croptopia 2.0.5
			com_typesafe_config: config 1.4.1
			io_leangen_geantyref_geantyref: geantyref 1.3.11
			org_spongepowered_configurate-core: configurate-core 4.1.2
			org_spongepowered_configurate-hocon: configurate-hocon 4.1.2
		decorative_blocks: Decorative Blocks 2.1.1
		doubledoors: Double Doors 4.5
		dungeons_arise: When Dungeons Arise 2.1.50
		eatinganimationid: Eating Animation 1.8
		ecologics: Ecologics 1.7.11
		edenring: EdenRing 0.5.2
		elementa: Elementa 590
		enchdesc: EnchantmentDescriptions 10.0.12
		essential: Essential 12666+deploy-staging+g0446b7ab4
		essential-container: essential-container 1.0.0
			essential-loader: essential-loader 1.0.0
		essential-dependencies: Essential Dependencies 0
			fabric-language-kotlin: Fabric Language Kotlin 1.8.2+kotlin.1.7.10
				org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.7.10
				org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.7.10
				org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.7.10
				org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.7.10
				org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.18.2
				org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.6.4
				org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.6.4
				org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
				org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.3.3
				org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.3.3
				org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.3.3
		fabric-api: Fabric API 0.75.1+1.18.2
			fabric-api-base: Fabric API Base 0.4.5+64b7c69360
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.3+2373a54560
			fabric-biome-api-v1: Fabric Biome API (v1) 8.0.6+651bcc5960
			fabric-block-api-v1: Fabric Block API (v1) 1.0.0+ee3621fc60
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.13+3ac43d9560
			fabric-command-api-v1: Fabric Command API (v1) 1.1.10+d7c144a860
			fabric-commands-v0: Fabric Commands (v0) 0.2.9+b4f4f6cd60
			fabric-containers-v0: Fabric Containers (v0) 0.1.24+d7c144a860
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.1.2+c85f2e3860
			fabric-convention-tags-v1: Fabric Convention Tags 1.0.2+474e4f7960
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.1+ee261a3e60
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 3.0.3+9fcb91d260
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.21+489d6f8260
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.9+d7c144a860
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.20+d7c144a860
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.21+d7c144a860
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.13+d7c144a860
			fabric-item-api-v1: Fabric Item API (v1) 1.6.1+a6f01b2660
			fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.15+91896a4960
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.12+54e5b2ec60
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.10+b4f4f6cd60
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.1.1+cc71601c60
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.1+1997ad1260
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.3+e747827960
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.5+cc71601c60
			fabric-models-v0: Fabric Models (v0) 0.3.7+d7c144a860
			fabric-networking-api-v1: Fabric Networking API (v1) 1.0.24+2d30af4b60
			fabric-networking-v0: Fabric Networking (v0) 0.3.11+b4f4f6cd60
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 2.1.4+032c981d60
			fabric-particles-v1: Fabric Particles (v1) 0.2.13+526dc1ac60
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.12+bb2e047760
			fabric-renderer-api-v1: Fabric Renderer API (v1) 0.5.0+bf48649860
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.6.1+03e2b68760
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.12+b4f4f6cd60
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.9+ee3621fc60
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 2.0.3+54e5b2ec60
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.14+b4f4f6cd60
			fabric-rendering-v1: Fabric Rendering (v1) 1.10.8+54e5b2ec60
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.0.3+91b7aa6660
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.5.2+446e059560
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.11+d882b91560
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.2.3+1f6558e860
			fabric-textures-v0: Fabric Textures (v0) 1.0.13+3ac43d9560
			fabric-transfer-api-v1: Fabric Transfer API (v1) 1.6.4+f4563ac860
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.1.0+e747827960
		fabricloader: Fabric Loader 0.14.17
		feature_nbt_deadlock_be_gone: Feature NBT Deadlock Be Gone - Fabric 2.0.0+1.18.2-rc1
		fpsreducer: FPS Reducer 1.18.2-2.0
		geckolib3: Geckolib 3.0.80
			com_eliotlash_mclib_mclib: mclib 20
		immersive_armors: Immersive Armors 1.5.5+1.18.2
		inventorio: Inventorio 1.7.4
		jade: Jade 5.99.10
		java: OpenJDK 64-Bit Server VM 17
		journeymap-fabric: Journeymap 5.9.4
			journeymap-api-fabric: JourneyMap API 1.18.2-1.9-fabric-SNAPSHOT
		libraryferret: Library ferret 4.0.0
		mca: Minecraft Comes Alive 7.4.0+1.18.2
		mcwbridges: Macaw's Bridges 2.0.7
		mcwdoors: Macaw's Doors 1.0.7
		mcwfences: Macaw's Fences and Walls 1.0.6
		minecraft: Minecraft 1.18.2
		modid: Creatures Of The Snow 2.1.0+1.18.2
		modmenu: Mod Menu 3.2.5
		more_gems: More Gems 1.3.72
			pugh_tools: Pugh Tools 1.2.7
		mythicmounts: Mythic Mounts 1.18.2-7.1
		optifabric: OptiFabric 1.13.16
			mm: Manningham Mills 2.3
		org_jetbrains_annotations: annotations 13.0
		patchouli: Patchouli 1.18.2-71-FABRIC
			fiber: fiber 0.23.0-2
		pfm: Paladin's Furniture 1.1.1
		probablychests: Probably Chests 0.5.5-1.18.2
		repurposed_structures: Repurposed Structures 4.1.15+1.18.2
			omega-config: OmegaConfig 1.0.8
		simple_backpack: SimpleBackpack 1.2.37
		simply_houses: Simply Houses 1.1.0
		skinlayers: 3d Skin Layers 1.5.2-mc1.18.2
		skylorlib: SkyLib 1.5.1-1.18.2
		smoothswapping: Smooth Swapping 0.6.1
		strawgolem: Straw Golem 2.0.0b2
		t_and_t: Towns and Towers 1.10.0.1
			blue_endless_jankson: jankson 1.2.1
		terrablender: TerraBlender 1.2.0.126
			com_electronwill_night-config_core: core 3.6.4
			com_electronwill_night-config_toml: toml 3.6.4
		tipsmod: Tips 5.0.11
		trinkets: Trinkets 3.3.1
		universal-graves: Universal Graves 2.0.3+1.18.2
			fabric-permissions-api-v0: fabric-permissions-api 0.1-SNAPSHOT
			hologram-api: Hologram API 0.2.1+1.18-pre5
			placeholder-api: Placeholder API 1.1.3+1.17.1
			polymer: Polymer (Core) 0.2.0-rc.2+1.18.2
				packet_tweaker: Packet Tweaker 0.3.0+1.18.2
				polymer-legacy: Polymer (Compatibility) 0.1.9+0.2.0-rc.2+1.18.2
			server_translations_api: Server Translations API 1.4.12+1.18.2
			sgui: sgui 1.0.2+1.18.2
		universalcraft: UniversalCraft 266
		vigilance: Vigilance 284
		yungsapi: YUNG's API 1.18.2-Fabric-2.2.9
			org_javassist_javassist: javassist 3.28.0-GA
		yungsextras: YUNG's Extras 1.18.2-Fabric-2.1.0
	Launched Version: fabric-loader-0.14.17-1.18.2
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: Unknown
	Window size: <not initialized>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: <disabled>
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: <unknown>
	OptiFine Version: OptiFine_1.18.2_HD_U_H9_pre1
	OptiFine Build: 20220823-161442
	Shaders: null
	OpenGlVersion: null
	OpenGlRenderer: null
	OpenGlVendor: null
	CpuCount: 0

-- OptiFabric --
Details:
	OptiFine jar designed for: 1.18.2
	OptiFine jar version: OptiFine_1.18.2_HD_U_H9_pre1
	OptiFine jar status: Valid OptiFine installer
	OptiFine remapped jar: C:/Users/Usuario/AppData/Roaming/.minecraft/essential_mod/fabric/1.18.2/1.18.2 Fabric Essential/.optifine/OptiFine_1.18.2_HD_U_H9_pre1/Optifine-mapped.jar
	OptiFabric error: <None>

my mods here

![image](https://github.com/FabricMC/fabric/assets/134734628/550a5d11-d218-4546-846d-52e7658416e0)

**Comment by warjort** [comment]
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method polymer_changeArmorTexture(Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;Lnet/minecraft/class_1309;Lnet/minecraft/class_1304;ILnet/minecraft/class_572;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;Lnet/minecraft/class_1799;)V in polymer.mixins.json:client.compat.armor_ArmorFeatureRendererMixin from mod polymer failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap **_polymer_**-refmap.json

Issue with the polymer mod.
`polymer: Polymer (Core) 0.2.0-rc.2+1.18.2`

You need to contact the mod author.
https://github.com/Patbox/polymer

But since this a problem with renderering, It might be a conflict with optifine?
So try without optifine installed.
